### PR TITLE
Add some n00b content to REAMDE

### PR DESCRIPTION
**Pull request by dsyer**
Make it easier to get started quickly by providing some pictures
and links to scripts and docker images to get the system running.

### CODE REVIEW

**Comment by eirslett** [comment]
Thanks, that looks great! +1

File: README.md [hunk 1]
```diff
@@ -2,7 +2,41 @@
...
+[Docker Zipkin](https://github.com/openzipkin/docker-zipkin) project,
+which (in addition to being able to build docker images) provides
+scripts and a
+[`docker-compose.yml`](https://github.com/openzipkin/docker-zipkin/blob/master/deploy/docker-compose.yml)
+for launching pre-built images, e.g.
+
+```
+$ git clone https://github.com/openzipkin/docker-zipkin
+$ cd docker-zipkin/deploy
+$ docker-compose
```
Comment (abesto) [inline README.md]: Maybe this should be `docker-compose up`?

**Comment by abesto** [comment]
Thanks a lot for writing this up, and also the awesome graph :) Definitely +1

File: README.md (hunk 1 above)
Comment (dsyer) [inline README.md]: Good catch. Fixed thanks.

**Comment by jamescway** [comment]
:+1: looks great

**Pull request merged**